### ISSUE TITLE
Not reliable for Brazil

### ISSUE THREAD

**Issue by jesobreira** [prograhammer/countries-regions-cities#4]
This database is not reliable for Brazil. It adds several cities to Sao Paulo, such as Vitoria and Vila Velha, that are actually in Espirito Santo. These are also listed in Espirito Santo so they are duplicated (and in the wrong region).

I can't confirm the full list of wrong cities, however it's interesting that the cities database brings 48k rows while it's known up to 38k cities in the world.

**Comment by prograhammer** [comment]
Yeah, it's due for a refresh real soon. It's on my todo list, I've just got some software about to lauch holding me back for a bit longer. I plain to go back to the US Geological service directly and update the data. This will remove the MaxMind disclosure as well. I'll ping you then! One tip though, is most companies now are just letting user's write in the city. If it's misspelled then later some staff member may query these (unique city records) and clean them up. This can be a lot less hassle depending on what your software needs.
